docs/docs/07-forms.md

```diff
@@ -166,3 +166,44 @@ To make an uncontrolled component, `defaultValue` is used instead.
 > Note:
 >
 > You can pass an array into the `value` attribute, allowing you to select multiple options in a `select` tag: `<select multiple={true} value={['B', 'C']}>`.
+
+### Submit form
+
+If you need to programmatically submit a form, you have to obtain a [reference to the DOM node](08.1-more-about-refs.md), and manually call `submit()` on it.
+
+```javascript
+var MyComponent = React.createClass({
+  handleClick: function() {
+    if (this.myForm !== null) {
```

Comment:
I don’t think it would ever be `null` here. If the ref is null, the component was unmounted, so you can’t possibly get into the event handler.